Frontend/src/pages/Voting-Overview.tsx

```diff
@@ -15,20 +20,79 @@ function VotingOverview() {
   // eligible voting count
   // more?
 
-  // useEffect(() => {
-  //   async function loadDAOInfo() {
-  //     if (daoContract && !currentWinningProposal) {
-  //       console.dir(daoContract)
-  //       const winningProposal = await daoContract.getWinningProposal()
-  //       console.log("Current winning proposal:", winningProposal.toString())
-  //       setCurrentWinningProposal(winningProposal.toString())
-  //     }
-  //   }
+  const totalVotes = proposals.reduce((acc, inc) => {
+    acc += inc.voteCount?.toNumber()
+    return acc
+  }, 0)
 
-  //   loadDAOInfo()
-  // }, [daoContract, currentWinningProposal])
+  return isProposalsLoading ? (
+    <Spinner />
+  ) : proposals.length > 0 ? (
+    <ContentWrapper>
+      <Flex flexDir="column">
+        <Heading as="h2">Voting Overview</Heading>
 
-  return <Box>Current winning proposal: {currentWinningProposal}</Box>
+        <Flex mt={10}>
+          <Text>Voting active? </Text>
+          <Text ml={5}>
+            {proposals.some((proposal) => !proposal.active)
+              ? "Finished"
+              : "Ongoing"}
+          </Text>
+        </Flex>
+
+        <Text mt={5}>Total Votes: {totalVotes}</Text>
+
+        <Flex
+          mt={10}
+          alignItems="center"
+          justifyContent="center"
+          w="full"
+          mb={4}
+        >
+          <Grid
+            templateRows="repeat(2, 1fr)"
+            templateColumns="repeat(2, 1fr)"
+            gap={4}
+            height="fit-content"
+            justifyContent="center"
+          >
+            {proposals.map((proposal, idx) => {
+              const { index, voteCount, collectionSize, active } = proposal
+
+              return (
+                <GridItem
+                  key={`proposal-${idx}`}
+                  flexDirection="column"
+                  border="2px"
+                  borderRadius="10px"
+                  p={10}
+                  bg={
+                    winningProposalIndex?.toString() === index?.toString()
+                      ? "green.100"
+                      : ""
+                  }
+                >
+                  <Text>Index: {index?.toString()}</Text>
+                  <Text>VoteCount: {voteCount?.toString()}</Text>
+                  <Text>
+                    Vote %:{" "}
+                    {totalVotes?.toString() === "0"
+                      ? "0"
+                      : voteCount?.toNumber() / totalVotes?.toNumber()}
+                  </Text>
+                  <Text>Collection Size: {collectionSize?.toString()}</Text>
+                  <Text>Is active: {active?.toString()}</Text>
```

Comment:
I'd remove this line since this voting round is finished altogether (it's not possible to keep voting)
Also, it's not the collections that are active or not, but the voting rounds. This is the logic that I'm implementing in my PR

Reply:
Ok, actually I just wanted to show the metrics of the last voting round.

With the current implementation we only have one, right? Or will it be possible to have the next one, once one finishes?

I just saw on the Proposal struct the `active` field, which is turned to `false` on the winningProposal which is my indicator that the voting has finished.

How should I do it with your implementation? Which field can I check?

Reply:
Sorry, got it. Is removed. The logic though, checking through all proposals for an `false` active field remains unless that logic changes with your implementation

Reply:
Maybe it is too late to add the rounds logic. I'm thinking opening another PR just adding a simple addProposal (collection) and that's it. Let's see.

But yeah, it's better to have that line removed, since it is confusing to think of a collection as active or not